src/store/ChartSettingStore.js

```diff
@@ -116,10 +120,14 @@ export default class ChartSettingStore {
         return this.language ? this.language : 'en';
     }
 
+    @action.bound setTheme(value) {
+        this.theme = value ? 'dark' : 'light';
+        this.mainStore.chart.stxx.clearStyles();
+        this.saveSetting();
+    }
 
-
-    @action.bound setView(view) {
-        this.view = view ? view : '';
+    @computed get isDarkTheme() {
+        return this.theme === 'dark';
```

Comment:
this is just nitpicking but... it's a one line function that's only used in one place. You could just pass this condition to the `<Switch />`

Reply:
@shayanbinary actually never mind this.